backend/src/indexer/block_processor.rs

```diff
@@ -149,6 +170,8 @@ impl ProcessEvent for BlockProcessor {
         PreparedBlock::batch_save(batch, &mut new_context, &mut tx).await?;
         for block in batch {
             block.process_block_content(&mut tx).await?;
+            self.last_processed_block_height.set(block.height);
+            self.last_processed_block_slot_time.set(block.slot_time.timestamp());
```

Comment:
I think we should do this right after the line with `tx.commit()` since it might reverse a bit again if something fails during the processing.